src/ethereum_test_specs/blockchain.py

```diff
@@ -652,7 +654,7 @@ def make_fixture(
             blocks=fixture_blocks,
             last_block_hash=head,
             pre=pre,
-            post_state=alloc,
+            post_state=alloc if not self.exclude_post_state else {},
```

Comment:
Initially, I did it with `None` instead of `{}`. This means the field didn't exist in the output.

While Geth was okay with it, Reth now is a bit more strict and find that problematic since it always expects the field to be set.

So the less invasive way is always have the field, but empty.